Source/Core/Core/FifoPlayer/FifoRecordAnalyzer.cpp

```diff
@@ -227,78 +228,3 @@ void FifoRecordAnalyzer::WriteVertexArray(int arrayIndex, u8 *vertexData, int ve
 
 	FifoRecorder::GetInstance().WriteMemory(arrayStart, arraySize, MemoryUpdate::VERTEX_STREAM);
 }
-
-void FifoRecordAnalyzer::ProcessTexMaps()
```

Comment:
They don't interact badly with each other, so it will be fine to merge this as is.